actor/hashing/hashring/hashring.go

```diff
@@ -0,0 +1,30 @@
+package hashring
+
+import (
+	"github.com/AsynkronIT/gam/actor"
+	"github.com/serialx/hashring"
```

Comment:
I didn't know about this lib,
It should be possible to use this to replace the home baked hashring I do here, right?
https://github.com/AsynkronIT/gam/blob/dev/cluster/registry.go#L28